test/Microsoft.AspNet.DataProtection.Test/Managed/ManagedAuthenticatedEncryptorTests.cs

```diff
@@ -77,7 +79,8 @@ public void Encrypt_Decrypt_Tampering_Fails()
             });
         }
 
-        [Fact]
+        [ConditionalFact]
+        [ConditionalRunTestOnlyOnWindows]
```

Comment:
To be honest, I'm not sure. I tried getting the same results on Linux but had no success on that. Then I noticed that there are tests with the same name in other classes and those were already marked to run on Windows only.